src/solution.rs

```diff
@@ -28,7 +32,7 @@ fn run_testcase<'a>(test: &'a TestCase, run_command: &mut Command, timeout: &Dur
         Err(error) => {
             let program = run_command.get_program().to_str().unwrap_or("Unable to run command");
```

Comment:
It's a rare case regardless, but would [`to_string_lossy`](https://doc.rust-lang.org/std/ffi/os_str/struct.OsStr.html#method.to_string_lossy) result in a better message in `Err` cases?

Reply:
Yeah that would be better.